functions/postgre-sql-python/handlers/handler.py

```diff
@@ -0,0 +1,56 @@
+import psycopg2
+from psycopg2 import Error
+import os
+import logging
+
+PG_HOST=os.environ.get('PG_HOST')
+PG_USER=os.environ.get('PG_USER')
+PG_DATABASE=os.environ.get('PG_DATABASE')
+PG_PASSWORD=os.environ.get('PG_PASSWORD')
+PG_PORT=os.environ.get('PG_PORT')
+PG_SSL_ROOT_CERT=os.environ.get('PG_SSL_ROOT_CERT')
```

Comment:
```suggestion
PG_HOST=os.environ["PG_HOST"]
PG_USER=os.environ["PG_USER"]
PG_DATABASE=os.environ["'PG_DATABASE"]
PG_PASSWORD=os.environ["PG_PASSWORD"]
PG_PORT=os.environ["'PG_PORT"]
PG_SSL_ROOT_CERT=os.environ["PG_SSL_ROOT_CERT"]
```

I prefer `os.getenv`  instead of `os.environ.get`. But because here those env vars are required, it's even better to catch the error early.